common/src/app/common/types/shape/interactions.cljc

```diff
@@ -87,7 +87,7 @@
     :close-overlay
     :prev-screen
     :open-url})
-
+(s/def ::position-relative-to (s/nilable ::us/uuid))
```

Comment:
You also need to add the new attribute to `action-opts-spec`, in the `:req-un` block of `:open-overlay` and `close-overlay`

Reply:
Yes. Or more exactly it should be on opt-un for backwards compatibility